crypto/ec/ec_lib.c

```diff
@@ -1007,3 +1007,21 @@ int ec_group_simple_order_bits(const EC_GROUP *group)
         return 0;
     return BN_num_bits(group->order);
 }
+
+/*-
+ * Coordinate blinding for EC_POINT.
+ *
+ * The underlying EC_METHOD can optionally implement this function:
+ * underlying implementations should return 0 on errors, or 1 on
+ * success.
+ *
+ * This wrapper returns 1 in case the underlying EC_METHOD does not
+ * support coordinate blinding.
+ */
+int ec_point_blind_coordinates(const EC_GROUP *group, EC_POINT *p, BN_CTX *ctx)
+{
+    if (group->meth->blind_coordinates == NULL)
+        return 1; /* ignore if not implemented */
+
+    return group->meth->blind_coordinates(group, p, ctx);
+}
```

Comment:
This is side note. [And "side note" means that no action is actually expected.] I wonder if you considered placing this function as static inline into ec_lcl.h instead. I mean since it's not public and is trivial, then why not have compiler inline it?

Reply:
@dot-asm This is a great suggestion. Since this is a backport from the 1.1.1 merged PR, a change here would require a new PR for 1.1.1?

Reply:
Yes. I don't recall if it was explicitly specified that we never re-open merged requests, but I don't recall one being re-opened either. It's only natural to view merged request as completed, and consequently it's only natural that further amendments are viewed as new/separate matter. So yes, it would take new request for master/1.1.1.